### PR TITLE
Add forceDownload

### PR DESCRIPTION
**Pull request by jeffhappily**

### CODE REVIEW

File: src/Network/HTTP/Download/Verified.hs [hunk 1]
```diff
@@ -243,9 +243,10 @@ verifiedDownload
          :: HasTerm env
          => DownloadRequest
          -> Path Abs File -- ^ destination
+         -> Bool -- ^ whether to redownload or not if file exists
```
Comment (snoyberg) [inline src/Network/HTTP/Download/Verified.hs]: Can this be added to `DownloadRequest` to avoid breaking the API here?

Comment (snoyberg) [reply src/Network/HTTP/Download/Verified.hs]: Actually looks like it might still be an API breakage, but still nicer than a plain `Bool`.

**Review by snoyberg — CHANGES_REQUESTED** [review]

File: src/Network/HTTP/Download/Verified.hs (hunk 1 above)
Comment (jeffhappily) [reply src/Network/HTTP/Download/Verified.hs]: Yup sure

**Comment by snoyberg** [comment]
Great! This should be a major version bump, so please:

* bump the version to 0.2.0.0
* add a ChangeLog entry

**Review by snoyberg — APPROVED** [review]

**Review by snoyberg — APPROVED** [review]

**Pull request merged**